flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/AvroRecordClassConverter.java

```diff
@@ -73,9 +75,37 @@ private AvroRecordClassConverter() {
 			final GenericTypeInfo<?> genericTypeInfo = (GenericTypeInfo<?>) extracted;
 			if (genericTypeInfo.getTypeClass() == Utf8.class) {
 				return BasicTypeInfo.STRING_TYPE_INFO;
+			} else if (genericTypeInfo.getTypeClass() == Map.class) {
```

Comment:
If you update this converter class, you should also update the corresponding runtime classes in `org.apache.flink.formats.avro.AvroRow(De)SerializationSchema`

Reply:
could you explain more about this? I didn't find any coupling between AvroRecordClassConverter and AvroRow(De)SerializationSchema. But I did encounter "UTF8<->String" cast problem during my integration which I was not sure if I should open a separate issue.